threejs/fxhash_Rooms/src/index.js

```diff
@@ -104,7 +97,35 @@ plane.position.y = figureMap.plane.pos.y;
 
 plane.receiveShadow = true;
 
-scene.add(sphere, plane, box);
+scene.add(plane);
+
+/**
+ * Constructive Solid Geometry Tests
+*/
+// let meshA = new THREE.Mesh(new THREE.BoxGeometry(1,1,1), material);
+// let meshB = new THREE.Mesh(new THREE.BoxGeometry(1,1,1), material);
+
+const meshA = new THREE.Mesh(
+  new THREE[figureMap.box.geometry.type](...figureMap.box.geometry.args),
+  material
+);
+const meshB = new THREE.Mesh(
+  new THREE[figureMap.box.geometry.type](...figureMap.box.geometry.args),
+  material
+);
+
+meshB.position.add(new THREE.Vector3( 0.5, 0.5, 0.5));
+
+meshA.updateMatrix();
+meshB.updateMatrix();
+
+let bspA = CSG.fromMesh( meshA );
+let bspB = CSG.fromMesh( meshB );
+
+let bspResult = bspA.union(bspB);
+let meshResult = CSG.toMesh( bspResult, meshA.matrix, meshA.material );
+
+scene.add(meshResult); // <---- anaglyphic does this work for anaver.se integration?
```

Comment:
Question for you @Anaglyph-ic, do you think this will work with anaver.se integration? I'm looking to union several planes with different locations into a single mesh.  (note the process found in issue 1 [here](https://github.com/michaelhoehn/fxHash-collabs/issues/1))

I'm seeing that anaver.se figures are primitives, is this the right approach? 

![image](https://user-images.githubusercontent.com/54593392/183201436-410840e1-3b56-4a7a-96a1-21601261fd04.png)

Reply:
note: the geometry above is just for demo purposes, the actual unioned geometry will be mesh planes.

Reply:
You **can't** have actual union/excusion at the moment, however you **can** have overlap between geometries. You can also have parametric geometries and extruded geometries. 

So for your Issue 1 use case, the cleanest way to go at it is to create an extrude geometry for the floor, and then add boxes for the walls. 

The anaver.se has a specific API for [extrude geometries](https://threejs.org/docs/?q=extru#api/en/geometries/ExtrudeGeometry). 

Rather than putting up a fake example, a commit a floor shape generator and I will add it so you have an idea of how it looks like.

The boxes can be added and placed like regular THREE geometries.